src/main/java/org/broadinstitute/hellbender/utils/codecs/gencode/GencodeGtfFeature.java

```diff
@@ -108,23 +109,24 @@ protected GencodeGtfFeature(final String[] gtfFields) {
         // But we need to match up the field names to the fields themselves:
         for ( final String extraField : extraFields ) {
 
-            final String[] fieldParts = extraField.trim().split(EXTRA_FIELD_KEY_VALUE_SPLITTER);
-
-            if ( fieldParts.length == 1 ){
-                if ( fieldParts[EXTRA_FIELD_KEY_INDEX].isEmpty() ){
-                    continue;
-                }
-                else {
-                    throw new UserException.MalformedFile("Extraneous optional field data - not in a key/value pair: " + extraField);
-                }
+            final String trimmedExtraField = extraField.trim();
+            if (trimmedExtraField.isEmpty()) {
+                continue;
             }
 
-            // Each optional field is in a key/value pair:
-            final String fieldName = fieldParts[EXTRA_FIELD_KEY_INDEX].trim();
-
-            // The value of the field may be between two quotes.
-            // We remove them here.
-            final String fieldValue = fieldParts[EXTRA_FIELD_VALUE_INDEX].trim().replaceAll("\"", "");
+            final int splitPoint = trimmedExtraField.indexOf(EXTRA_FIELD_KEY_VALUE_SPLITTER);
+            final String fieldName;
+            final String fieldValue;
+            if(splitPoint == -1 ) {
+                throw new UserException.MalformedFile("Extraneous optional field data - not in a key/value pair: " + extraField);
+            } else {
+                fieldName = trimmedExtraField.substring(0,splitPoint).trim();
+
+                // The value of the field may be between two quotes.
+                // We remove them here.
+                final String rawFieldValue = trimmedExtraField.substring(splitPoint + 1, trimmedExtraField.length());
+                fieldValue = StringUtils.remove(rawFieldValue.trim(), '"');
```

Comment:
Can we assume that the quotes are always at the beginning/end? If yes, we could make this even faster using a `substring()` call.

Reply:
@lbergelson - I think we can assume that quotes would wrap the strings.

That said, we will need to check to see if quotes are in the string before blindly substringing them.